pkg/tracing/config.go

```diff
@@ -1,182 +0,0 @@
-// Copyright Istio Authors
```

Comment:
There were some API changes that broke here. Its dead code though, so I just removed instead of fixing it

Reply:
I hadn't realized that this was no longer used, but it makes sense with Mixer no longer being around.